sdk/Lusid.Sdk.Tests/Utilities/TestDataUtilities.cs

```diff
@@ -541,17 +541,16 @@ public static UpsertRecipeRequest BuildRecipeRequest(string recipeCode, string s
         /// Helper function to perform valuation of instrument. Can either perform inline valuation or valuation through a portfolio.
         /// If it is through a portfolio then a new portfolio and transaction against the instrument is created which is deleted at the end. 
         /// </summary>
-        public static ValuationRequest ValuationRequest(LusidInstrument instrument,  string scope, ModelSelection.ModelEnum model, DateTimeOffset effectiveAt, string recipeCode, string portfolioCode)
+        public static ValuationRequest ValuationRequest(string scope, DateTimeOffset effectiveAt, string recipeCode, string portfolioCode)
```

Comment:
Consider changing the name of these functions so that they don't have the exact same name as their returning type. Maybe prefix with Create, Get, Build, etc